tests/e2e/package.json

```diff
@@ -7,58 +7,27 @@
-    "test-oauth": "tsc && mocha --config mocha.ocp.link.json",
-    "test-intelij": "tsc && mocha --config mocha.intelij.json",
-    "test-plugin": "./initPluginTest.sh",
-    "test-all-devfiles": "./generateIndex.sh && ./initDevfileTests.sh",
-    "test-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-theia.json",
-    "load-test-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-load-theia.json",
-    "test-happy-path-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-happy-path-theia.json",
-    "test-devworkspace-happy-path-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-devworkspace-happy-path-theia.json",
-    "test-ws-creation-and-ls-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-ws-creation-and-ls-theia.json",
-    "test-java-vertx-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-java-vertx-theia.json",
-    "test-java-springboot-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-java-springboot-theia.json",
-    "test-git-ssh-theia": "./generateIndex.sh &&  npm run lint && npm run tsc && mocha --config mocha-git-ssh-theia.json",
-    "test-git-self-sign-cert-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-git-self-sign-cert-theia.json",
-    "test-git-publish-branch-theia": "./generateIndex.sh &&  npm run lint && npm run tsc && mocha --config mocha-git-publish-branch-theia.json",
-    "test-openshift-connector-theia": "./generateIndex.sh &&  npm run lint && npm run tsc && mocha --config mocha-connector-theia.json",
-    "test-factory-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-factory-theia.json",
-    "test-all-factories-theia": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-all-factories-theia.json",
-    "test-plugin-ci-theia": "export TS_DELETE_PLUGINS_TEST_WORKSPACE=true && npm run init-mocha-opts -- --spec dist/tests/plugins/theia/${USERSTORY}.spec.js",
-    "test-all-plugins-theia": "tsc && mocha --config mocha-all-plugins-theia.json",
-    "test-code": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-che-code.json",
-    "test-happy-path-code": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-happy-path-che-code.json",
-    "test-devworkspace-happy-path-code": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-devworkspace-happy-path-che-code.json",
-    "test-empty-workspace-devworkspace-happy-path-code": "./generateIndex.sh && npm run lint && npm run tsc && mocha --config mocha-empty-workspace-devworkspace-happy-path-che-code.json",
```

Comment:
@nallikaea : this command is used by che-operator PR check https://github.com/eclipse-che/che-operator/blob/main/build/scripts/oc-tests/oc-test-happy-path.sh#L26

If you are going to remove it, please prepare PR to https://github.com/eclipse-che/che-operator with alternative command.

Reply:
https://github.com/eclipse-che/che-operator/pull/1641

Reply:
Please, don't remove `test-empty-workspace-devworkspace-happy-path-code` until the PR is merged